src/Examples/Flowsheets/methanol_synthesis.ipynb

```diff
@@ -201,6 +201,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "WARNING: DEPRECATED: The default argument for the ProcessBlock class is\n",
```

Comment:
This appears to have been run with an older version of IDAES - this warning has been fully deprecated now, so we should re-run the notebook using the latest IDAES main.